tensorflow-framework/src/test/java/org/tensorflow/framework/activations/ELUTest.java

```diff
@@ -0,0 +1,103 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=======================================================================*/
+package org.tensorflow.framework.activations;
+
+import org.junit.jupiter.api.*;
+import org.tensorflow.Operand;
+import org.tensorflow.framework.utils.TestSession;
+import org.tensorflow.op.Ops;
+import org.tensorflow.types.TFloat32;
+import org.tensorflow.types.TFloat64;
+import org.tensorflow.types.TInt32;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/** @author Jim Clarke */
+public class ELUTest {
+
+  private final TestSession.Mode[] tfModes = {TestSession.Mode.EAGER, TestSession.Mode.GRAPH};
+
+  public ELUTest() {}
+
+  @BeforeAll
+  public static void setUpClass() {}
+
+  @AfterAll
+  public static void tearDownClass() {}
+
+  @BeforeEach
+  public void setUp() {}
+
+  @AfterEach
+  public void tearDown() {}
+
+  /** Test of ELU call method */
+  @Test
+  public void testCallInt() {
+    for (TestSession.Mode tfMode : tfModes)
+      assertThrows(
+          java.lang.IllegalArgumentException.class,
+          () -> {
+            try (TestSession session = TestSession.createTestSession(tfMode)) {
+              Ops tf = session.getTF();
+              Operand<TInt32> input = tf.constant(new int[] {1, 2, 3, 4, 5});
+              ELU<TInt32> instance = new ELU<>(tf);
+              Operand<TInt32> result = instance.call(input);
+            }
+          });
+  }
+
+  /** Test of ELU call method */
+  @Test
+  public void testCallFloat() {
```

Comment:
Perhaps also consolidate this sort of test logic into a helper class? E.g.
~~~
assertFloat32Output(tf -> new ELU<Float32>(tf),
            {1f, -0.86466473f, 3f, -0.9816844f, -0.63212055f, 2f, -0.95021296f, 4f},
            {1, -2, 3, -4, -1, 2, -3, 4})